zql/zql.peg

```diff
@@ -287,41 +298,34 @@ fieldExprList
       RETURN(result)
   }
 
-countOp
-  = "count"i { RETURN("Count") }
-
-fieldReducerOp
-  = "sum"i  { RETURN("Sum") }
-  / "avg"i  { RETURN("Avg") }
-  / "stdev"i { RETURN("Stdev") }
-  / "sd"i   { RETURN("Stdev") }
-  / "var"i  { RETURN("Var") }
-  / "entropy"i { RETURN("Entropy") }
-  / "min"i  { RETURN("Min") }
-  / "max"i  { RETURN("Max") }
-  / "first"i  { RETURN("First") }
-  / "last"i  { RETURN("Last") }
-  / "countdistinct"i { RETURN("CountDistinct") }
-
-paddedFieldExpr = _? field:fieldExpr _? { RETURN(field) }
+ExprList
+  = first:Expression rest:(__ "," __ Expression)* {
+      VAR(result) = ARRAY(first)
 
-countReducer
-  = op:countOp _? "(" field:paddedFieldExpr?  _? ")" {
-    VAR(r) = MAP("op": op, "var": "count")
-    if ISNOTNULL(field) {
-      r["field"] = field
-    }
-    RETURN(r)
-  }
+      FOREACH(ASSERT_ARRAY(rest), r) {
+        APPEND(result, ASSERT_ARRAY(r)[3])
+      }
 
-fieldReducer
-  = op:fieldReducerOp _? "(" _? field:fieldExpr  _? ")" {
-    VAR(r) = MAP("op": op, "var": toLowerCase(op))
-    if ISNOTNULL(field) {
-      r["field"] = field
+      RETURN(result)
     }
-    RETURN(r)
-  }
+
+countOp
+  = "count" { RETURN("count") }
+
+ReducerOp
+  = ReducerOps { RETURN(TEXT) }
+
+ReducerOps
+  = "sum"
+  / "avg"
+  / "stdev"
+  / "var"
+  / "entropy"
+  / "min"
+  / "max"
+  / "first"
+  / "last"
+  / "countdistinct"
```

Comment:
Nit: Sort these.

Reply:
They are in the ordered by related functionality.   The procs are laid out this way too, grouped by like functionality rather than alphabetic.  I want to make this list go away in a later PR anyway (i.e., so anything of the form `| agg() ...` so we can add reducers without changing the grammarrs).